File: tuts/002-vpc-gs/python/ec2_wrapper.py
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

import logging
import boto3
from botocore.exceptions import ClientError

logger = logging.getLogger(__name__)


class EC2Wrapper:
    def __init__(self, client):
        self.client = client

    # TODO: Add wrapper methods matching the CLI tutorial actions

File: tuts/012-transitgateway-gettingstarted/swift/Sources/Ec2Wrapper.swift
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import AWSEc2
import Foundation

public class Ec2Wrapper {
    let client: Ec2Client

    public init(client: Ec2Client) {
        self.client = client
    }

    // TODO: Add async throws wrapper methods matching CLI tutorial actions
}
